### PR TITLE
Fix incorrect fragment identifier

### PR DESCRIPTION
**Pull request by FouoF**
### Description
Wrong bookmark format causes the bookmark text to be displayed incorrectly on the front-end page.
[递归服务器场景-recursive-server-scenarios](https://kubernetes.io/zh-cn/docs/concepts/cluster-administration/flow-control/#%E9%80%92%E5%BD%92%E6%9C%8D%E5%8A%A1%E5%99%A8%E5%9C%BA%E6%99%AF-recursive-server-scenarios)

### CODE REVIEW

**Review by windsonsea — COMMENTED** [review]
Thanks. No more in-synced content with the english upstream after checking:

- [English history](https://github.com/kubernetes/website/commits/main/content/en/docs/concepts/cluster-administration/flow-control.md)
- [Chinese history](https://github.com/kubernetes/website/commits/main/content/zh-cn/docs/concepts/cluster-administration/flow-control.md)

File: content/zh-cn/docs/concepts/cluster-administration/flow-control.md [hunk 1]
```diff
@@ -133,7 +133,7 @@ both the original request and some subsidiary ones(s), no matter how
 deep in the recursion, there is a danger of priority inversions and/or
 deadlocks.
 -->
-## 递归服务器场景     {#Recursive server scenarios}
+## 递归服务器场景     {#Recursive-server-scenarios}
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/cluster-administration/flow-control.md:136]:
```suggestion
## 递归服务器场景     {#recursive-server-scenarios}
```

**Comment by windsonsea** [comment]
Please [squash your commits into a single one](https://kubernetes.io/docs/contribute/new-content/open-a-pr/#squashing-commits).

**Review by sftim — COMMENTED** [review]
LGTM
/retitle Fix incorrect fragment identifier

**Pull request merged**